doc/source/conf.py

```diff
@@ -162,6 +162,8 @@
 edit_on_github_source_root = ""
 edit_on_github_doc_root = "docs"
 
+edit_on_github_skip_regex = '_.*|api/.*'
```

Comment:
I think this will work better if it's `generated` rather than `api`? Have you tested to see if this works as is?